HoloPipelines/core/services/np_image_manipulation.py

```diff
@@ -51,11 +51,11 @@ def downscale_and_conditionally_crop(
         x = image.shape[0]
```

Comment:
The FIXME comment in line 13 should be removed if you fixed it

Reply:
crop_around_centre should have a doc string and the arguments should have better names

Reply:
FIXME comment removed and arguments naming updated